rocky/rocky/views/tasks.py

```diff
@@ -115,3 +118,88 @@ def get_context_data(self, **kwargs):
         ]
 
         return context
+
+
+class AllTaskListView(ListView):
+    paginate_by = 100
+    paginator_class = RockyPaginator
+
+    # @staff_member_required
+    def get_queryset(self):
+        task_type = self.request.GET.get("type", self.plugin_type)
+        status = self.request.GET.get("scan_history_status") if self.request.GET.get("scan_history_status") else None
+        input_ooi = self.request.GET.get("scan_history_search") if self.request.GET.get("scan_history_search") else None
```

Comment:
```suggestion
        status = self.request.GET.get("scan_history_status", None)
        input_ooi = self.request.GET.get("scan_history_search", None)
```
or if you're doing a string emptiness check and a None check at the same time:
```suggestion
        status = self.request.GET.get("scan_history_status", "") or None
        input_ooi = self.request.GET.get("scan_history_search", "") or None
```